Tools/ardupilotwaf/boards.py

```diff
@@ -245,6 +267,8 @@ def configure_env(self, cfg, env):
             'AP_HAL_Linux',
         ]
 
+    def __init__(self):
+        self.with_uavcan = True
```

Comment:
Even when it is, this is something to be on a board level, not for all Linux boards. Please remove this.